ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -23,19 +24,29 @@
 """
 
 
+def ordered_representer(dumper, data):
+    return dumper.represent_mapping('tag:yaml.org,2002:map', data.items())
+
+yaml.add_representer(OrderedDict, ordered_representer, Dumper=yaml.SafeDumper)
```

Comment:
This might end up being useful elsewhere, so eventually we might want to move it to some kind of `utils` module.

Not asking you to do it now, just writing it down because it helps me remember. 😄